indexing-hadoop/src/test/java/io/druid/indexer/path/GranularityPathSpecTest.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Druid - a distributed column store.
+ * Copyright 2015 - Yahoo! Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.druid.indexer.path;
+
+import com.metamx.common.Granularity;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+
+public class GranularityPathSpecTest
+{
+  private GranularityPathSpec granularityPathSpec;
+  private final String TEST_STRING_PATH = "TEST";
+  private final String TEST_STRING_PATTERN = "*.TEST";
```

Comment:
These tests are pretty innocuous.  All they are doing at this point is verifying the setter, so the pattern provided doesn't really matter...